src/com_tjvendors/admin/sql/install.mysql.utf8.sql

```diff
@@ -36,34 +36,28 @@ PRIMARY KEY (`id`)
 
 CREATE TABLE IF NOT EXISTS `#__tjvendors_vendors` (
   `vendor_id` int(11) NOT NULL AUTO_INCREMENT,
-  `user_id` int(11) NOT NULL,
-  `vendor_title` varchar(255) NOT NULL,
+  `user_id` int(11) NOT NULL DEFAULT 0,
+  `vendor_title` varchar(255) NOT NULL DEFAULT '',
```

Comment:
`vendor_title` varchar(255) NOT NULL DEFAULT 
=>
`vendor_title` varchar(255) DEFAULT NULL

makes more sense

Reply:
Joomla has used NOT NULL DEFAULT '' for title fields hence used it